src/auditwheel/main_repair.py

```diff
@@ -92,6 +93,12 @@ def configure_parser(sub_parsers):
         help="Do not check for higher policy compatibility",
         default=False,
     )
+    p.add_argument(
+        "--add-path",
+        dest="PATHS",
+        default="",
+        help=f"Additional path(s) to search for libraries, {os.pathsep!r}-delimited",
+    )
```

Comment:
I understand that you wanted to use the same name as `delvewheel` which seems like a good thing however, it feels more like `-L`/`--library-path` or `-rpath-link` to mimic a bit more what feels like `ld` would be doing. I'd probably go with `--library-path`, allowed to be specified multiple times. Unfortunately, `-L` is already used for something else...

I'd be in favor of having the option specified multiple times with a single path each time rather than a single string option.

@lkollar, any thoughts on the name ? (Once it gets in, we won't be able to change it without breaking users).

Reply:
@lkollar you're not missing anything, this is accurate, it can indeed be achieved by manipulating LD_LIBRARY_PATH. I hadn't realized that the "repair wheel command" was invoked in a shell, making it possible to alter environment variables. Now I'm not sure whether we actually want this option..

Reply:
@jlaine I guess too many people do not realize that LD_LIBRARY_PATH is a solution for `ValueError: Cannot repair wheel, because required library "xxx.so" could not be located`, and that's exactly the reason why this PR is needed.

> Unfortunately, `-L` is already used for something else...

@mayeut and I can't say I didn't try it, because there was no other option.

`--add-path` is good for parity with `delvewheel` https://github.com/adang1345/delvewheel#additional-options

I would also add the pointer to it the error message, so that instead of plain `ValueError` there is a hint what to do next.

Reply:
Actually the proper fix is to propagate the option values to `lddtree` here. It already parses LD_LIBRARY_PATH, so no need to handle that.

https://github.com/pypa/auditwheel/blob/55bc5ee7de795819fccfd6468129c1041bcb9ac7/src/auditwheel/wheel_abi.py#L83